libs/core/hardware/include/hpx/hardware/timestamp/linux_generic.hpp

```diff
@@ -14,11 +14,20 @@
 
 namespace hpx { namespace util { namespace hardware {
 
+#if defined(HPX_HAVE_CUDA)
+    HPX_HOST_DEVICE
+#endif
```

Comment:
The `#if defined` around the macro is not necessary. That's why `HPX_HOST_DEVICE` is a macro.

Reply:
Aah, I thought it appends it all situations. I'll correct that.